scripts/setup/install.ps1

```diff
@@ -32,7 +32,11 @@ Param(
 
     [parameter()]
     [switch]
-    $DontCopy
+    $DontCopy,
+
+    [parameter()]
+    [switch]
+    $LegacyConfigurations
```

Comment:
Shouldn't this be default true.

Reply:
I use switch to be succinct. Since installed.ps1 is not meant to called directly, upstream would have populated this value anyway.

Reply:
My appologies, for some reason I though this was `setup.ps1` when I wrote that comment. I see it there now.